src/compat/get_webm_hdr_profile.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2015 CANAL+ Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { IHDRInformation } from "../parsers/manifest";
+
+/**
+ * Extract the webm HDR profile out of the codec string.
+ * The syntax of the codec string is defined in VP Codec ISO Media File Format
+ * Binding, in the section Codecs Parameter String.
+ * @param {string} codecString
+ * @returns {Object | undefined}
+ */
+export default function getWEBMHDRProfile(
```

Comment:
IMO this should not be in compat as this is not about browser compatibility

Putting it in the DASH manifest parser files makes more sense, no?